ui/src/app/pages/access/two-factor-auth-modal.spec.tsx

```diff
@@ -1,15 +1,15 @@
 import * as React from 'react';
 
-import { render, RenderResult, screen } from '@testing-library/react';
+import { RenderResult, screen } from '@testing-library/react';
+
+import { renderForModal } from 'testing/react-test-helpers';
 
 import { TwoFactorAuthModal } from './two-factor-auth-modal';
 
 describe('TwoFactorAuthModal', () => {
   const component = (): RenderResult =>
-    render(
-      <div id='popup-root'>
-        <TwoFactorAuthModal onCancel={() => {}} onClick={() => {}} />
-      </div>
+    renderForModal(
```

Comment:
added recently https://github.com/all-of-us/workbench/pull/7894/files#r1298720239